### PR TITLE
Top-align contributor

### PR DESCRIPTION
**Pull request by denektenina**
# Description

Link to Jira [Top-align contributor column](https://unit.atlassian.net/browse/NP-46796)

Move the alignment of the table rows from center to top. Also filled in the role field and added label to make it more similar to design, and match the number field. Also fixed alignment of the icons.

Before:

image

After:

image

# Checklist

Ensure that the changes are aligned with the expectations of PO/designer before marking the PR as ready for review.

Also ensure that the following criterias are met:

- [x] The changes are working as expected
- [x] The changes are tested OK for different screen sizes
- [x] The changes are tested OK for a11y
- [x] Interactive elements have data-testids
- [x] I have done a QA of my own changes

_Note: some of these criterias may not always be relevant, and can simply be marked as completed._

### CODE REVIEW

File: src/pages/registration/contributors_tab/components/ContributorRow.tsx [hunk 1]
```diff
@@ -118,7 +118,8 @@ export const ContributorRow = ({
               <TextField
                 {...field}
                 select
-                variant="standard"
+                variant="filled"
+                label={'Velg rolle'}
```
Comment (elkebab) [inline src/pages/registration/contributors_tab/components/ContributorRow.tsx:122]: i18n

File: src/pages/registration/contributors_tab/components/ContributorRow.tsx [hunk 2]
```diff
@@ -68,8 +68,8 @@ export const ContributorRow = ({
 
   return (
     <TableRow>
```
Comment (elkebab) [inline src/pages/registration/contributors_tab/components/ContributorRow.tsx:70]: Må ikke nødvendigvis gjøre dette, men hvis vi vet at alle celler skal være top-aligned kunne vi definert det her i stedet for hver enkelt celle
```suggestion
    <TableRow sx={{ td: { verticalAlign: 'top' } }}>
```

File: src/pages/registration/contributors_tab/components/ContributorRow.tsx (hunk 1 above)
Comment (denektenina) [reply src/pages/registration/contributors_tab/components/ContributorRow.tsx:122]: 😑 seff

File: src/pages/registration/contributors_tab/components/ContributorRow.tsx (hunk 2 above)
Comment (denektenina) [reply src/pages/registration/contributors_tab/components/ContributorRow.tsx:70]: Ja, kan like gjerne gjøre det tror jeg.

File: src/pages/registration/contributors_tab/components/ContributorRow.tsx [hunk 3]
```diff
@@ -118,7 +118,8 @@ export const ContributorRow = ({
               <TextField
                 {...field}
                 select
-                variant="standard"
+                variant="filled"
+                label={t('common.number_short')}
```
Comment (elkebab) [inline src/pages/registration/contributors_tab/components/ContributorRow.tsx:122]: feil key? :D 
```suggestion
                label={t('common.select_role')}
```

Comment (denektenina) [reply src/pages/registration/contributors_tab/components/ContributorRow.tsx:122]: At det går an :D

**Review by elkebab — APPROVED** [review]

**Pull request merged**